core/src/main/scala/chisel3/Printable.scala

```diff
@@ -65,7 +65,7 @@ object Printable {
 
   /** Pack standard printf fmt, args* style into Printable
     */
-  def pack(fmt: String, data: Data*): Printable = {
+  def pack(fmt: String, data: DataImpl*): Printable = {
```

Comment:
undo this maybe

Reply:
Yes please undo this, `DataImpl` is package private